src/components/Social/Header.js

```diff
@@ -46,14 +48,14 @@ const styles = {
     ...ellipsize
   }),
   subline: css({
-    ...sansSerifRegular14,
-    lineHeight: '20px',
+    ...convertStyleToRem(sansSerifRegular14),
+    lineHeight: pxToRem('20px'),
     color: colors.text,
     display: 'flex',
     alignItems: 'center'
   }),
   sublineText: css({
-    ...ellipsize
```

Comment:
whats problems did ellipsize cause? Generally I'd like to keep the subline ellipsized.